### ISSUE TITLE
Disable modal version of dropdown menu on mobile

### ISSUE THREAD

**Issue by ryantbrown** [rafaelpimpa/buefy#359]
When using a dropdown as a popover, which according to #113 and #103, is the recommended solution, I noticed that when clickable they are presented as a modal on mobile (hoverable dropdowns are presented normally).

Is there a way to disable the modal on mobile? I don't see anything in the documentation or the API regarding this.  It seems like this should be something that is configurable, that a modal would always work for a dropdown seems like a pretty big assumption.

Thanks in advance, if this is not currently supported would you accept a PR for it?

**Comment by jtommy** [comment]
@ryantbrown At the moment only `hovered` and `inline` props disable modal on mobile.
You can create a PR and we'll review it.

**Comment by lizhongyi** [comment]
me too....

**Comment by jtommy** [comment]
@lizhongyi There is already a prop `mobile-modal`.

**Comment by grebebird** [comment]
@jtommy - I was excited to see this issue was fixed, but I'm unfortunately not able to get it to work. I added **mobil-model="false"** to b-dropdown, but it still shows as a modal on smaller width screens. Have I made a dumb mistake or am I misunderstanding how it's supposed to work?

The way I recreated here was simply dragging the divider between the view and the code further to the left to simulate a small screen.
https://codepen.io/anon/pen/XZBGxG?editors=1010
